webapp/app/pods/components/commit-file/styles.scss

```diff
@@ -56,8 +64,86 @@
   margin-top: 15px;
 }
 
-.peekButton,
 .fileInput {
+	opacity: 0;
+	overflow: hidden;
+	position: absolute;
```

Comment:
The indent level seems off here — shouldn’t `stylelint` be failing?

Reply:
We don’t have `stylelint` in the project... This could be a nice contribution 😉 In the meantime, I will format the scss files with `prettier` so simple format error like this should make the build fail.